core/src/main/java/com/netease/arctic/table/TableMetaStore.java

```diff
@@ -542,7 +542,7 @@ public Builder withAuth(
         String authMethod, String hadoopUsername, byte[] krbKeyTabBytes, byte[] krbConfBytes,
         String krbPrincipal) {
       this.disableAuth = false;
-      this.authMethod = authMethod;
+      this.authMethod = authMethod == null ? null : authMethod.toUpperCase();
```

Comment:
I refer to add a statment after line 622 like this :
`authMethod  = authMethod.toUpperCase()`

And change line 609 to `String authMethod = properties.get(AUTH_METHOD).toUpperCase()`

How do you think?

Reply:
The reason for this suggestion is that I hope to keep `Builder` the ability to accpet upper/lower case parameters and the build method is responsible for converting the case of the parameter to the right one.

Reply:
The method `withKrbAuth` has the constant `AUTH_METHOD_SIMPLE/AUTH_METHOD_KERBEROS` assigned to it, so I think it's better not to have the value `kerberos` in `Builder`

> I hope to keep `Builder` the ability to accpet upper/lower case parameters

`withAuth` keep the ability to accpet upper/lower case parameters

Reply:
Make sense to me, but you may still need to change the `Builder#readProperties` method to make it accept lowercase properties value.

Reply:
done.